README.md

```diff
@@ -347,7 +347,7 @@ All props are optional.
 
 **text** { String } *Required if child is an element*: If `text` has a value, its first letter will be the letter a user can type to navigate to that item.
 
-**value** { String | Boolean | Number }: *Required if child is an element.* If `value` has a value, it will be passed to the `onSelection` handler when the `MenuItem` is selected.
+**value** { Object }: *Required if child is an element.* If `value` has a value, it will be passed to the `onSelection` handler when the `MenuItem` is selected.
```

Comment:
I think the value here could be `Any`, right? It's really whatever you want to pass to your `onSelection` callback?

Reply:
You're right. I've updated the PR 👍